tests/apps/test_similarity.py

```diff
@@ -222,142 +223,224 @@ def test_event_cardinality(photons, max_count, modes, expected):
     assert similarity.event_cardinality(photons, max_count, modes) == expected
 
 
+class TestGetState:
+    """Tests for the function ``strawberryfields.apps.similarity._get_state.``"""
```

Comment:
```suggestion
    """Tests for the function ``strawberryfields.apps.similarity._get_state``."""
```

Reply:
Actually this seems to happen a lot in this module, even before this PR, so not a big deal :shrug: